src/AbstractITILChildTemplate.php

```diff
@@ -35,6 +35,7 @@
 
 use Glpi\ContentTemplates\ParametersPreset;
 use Glpi\ContentTemplates\TemplateManager;
+use Glpi\Features\Clonable;
```

Comment:
What about making the whole `CommonDropdown` clonable?

Reply:
I agree, we could make them all clonable as we target main branch.

Reply:
That does mean that any dropdown in GLPI and all of its plugins have no choice but to be clonable. Some types may require extra work for cloning to work properly. I didn't think that would be desired.

Reply:
In another hand, why would some Dropdowns be clonable, and not others? It's difficult to explain why, even for us (and impossible to say which are not)

Reply:
If a plugin has a class that extends `CommonDropdown` with a UUID, secret, etc fields there would certainly need to be special handling during the cloning process. Most likely, a new UUID or secret would be generated. Having multiple items with the same UUID would likely lead to undefined behavior within the plugin.

If a dropdown has related data, they may need to specify that relation in `getCloneRelations` especially if that related information is required/usually created automatically when the main dropdown item is created.

For examples where we may not want them to be cloned at all, I can't see any real-life examples currently.

Reply:
Adding the trait and the `getCloneRelations` method is not a bid deal and, indeed, giving ability to clone any dropdown without any condition may result in issues in plugins.
Also, we did not have a lot of complains about the lack of clne feature on dropdowns, so I guess it will not be a problem to still have some dropdowns that are not clonable.